client/src/components/PosterList.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Poster from './Poster'
+
+PosterList.propTypes = {
+  list: PropTypes.array,
+}
+
+export default function PosterList({ list }) {
+  return (
+    <StyledPosterList>
+      {list.map(({ id, poster_path, name }) => {
+        return (
+          <li key={id}>
+            <Poster
+              path={
+                poster_path && `https://image.tmdb.org/t/p/w300/${poster_path}`
+              }
+              alt={`Poster von ${name}`}
```

Comment:
Isn't it a Cover?

Reply:
I'm using the wording from the api. They call it a poster.